app/views/students/status_updates/index.html.slim

```diff
@@ -15,13 +16,12 @@ table.table.table-striped.table-bordered.table-condensed
       th
 
   tbody
-    - @status_updates.each do |status_update|
+    - status_updates.each do |status_update|
       tr
-        td = l status_update.created_at
+        /date format is Weekday, MM dd, hh:mm
+        td = l status_update.created_at, :format => ('%A, %B %e, %H:%M (utc)')
```

Comment:
Also, we can use local time: every user can define a timezone in their user profile. The Rails app will set the individual time zone automatically or default to UTC.